DataFixtures/Dumper/YamlDataDumper.php

```diff
@@ -27,8 +27,7 @@ protected function transformArrayToData($data)
             $data,
             $inline = 3,
             $indent = 4,
-            $exceptionOnInvalidType = false,
-            $objectSupport = true
+            Yaml::DUMP_EXCEPTION_ON_INVALID_TYPE|Yaml::DUMP_OBJECT
```

Comment:
`Yaml::DUMP_EXCEPTION_ON_INVALID_TYPE` was not enabled before.

Reply:
ah `$exceptionOnInvalidType = false` you are right!